src/mousetracker.js

```diff
@@ -2879,6 +2879,12 @@
                 }
             }
 
+            // A primary mouse button may have been released while the non-primary button was down
+            if (pointsList.contacts > 0 && pointsList.type === 'mouse') {
+                // Stop tracking the mouse
+                pointsList.contacts--;
+                return true;
```

Comment:
These are good questions. I don't know that this best way to fix the issue and I want to invite discussion. Here are my thoughts: The return true is used to trigger the next function to call releasePointer, which decrements pointsList.captureCount by pointerCount || 1. I considered setting pointsList.contacts to 0 in my change, but I thought that might potentially conflict with releasePointer because it decrements pointList.captureCount rather than setting it to 0. It might be appropriate to instead call a panic function like abortTouchContacts that sets everything to 0 and resets everything... but that didn't seem necessary.

I assumed that we don't need to cover the case where a non-primary button is released while the primary button is down. If the primary button was pushed down first, the non-primary button press would never be registered by the browser. If the non-primary button was pressed first, it would hide the primary button press. Then, the non-primary button release would be lost, preventing it from running a non-primary release handler for it. I didn't see any issues that would arise from that since openseadragon doesn't remember the non-primary button clicks. (I'm also not sure we could detect the case because the non-primary clicks aren't remembered.) I guess that if an app added a non-primary release handler, they might see issues, but they could probably resolve it by clicking again.